lib/paths.cf

```diff
@@ -315,7 +315,7 @@ bundle common paths
       "path[tr]"            string => "/usr/bin/tr";
       "path[wc]"            string => "/usr/bin/wc";
       "path[wget]"          string => "/usr/bin/wget";
-      "path[realpath]"      string => "/usr/bin/realpath";
+      "path[realpath]"      string => "/usr/bin/readlink -e";
```

Comment:
Still it makes me slightly uncomfortable that we are kind of lying about the path to the binary. Its not really realpath, its really readlink.

And that would be OK if we were content to suffer on with the current way of doing things forever, but we would break it for people if we introduce actual PATHs for commands. Similar to how a few people like @zzamboni complained when we re-structured standard_services and some of the aliases like httpd => apache2 were not preserved.